code/controllers/configuration.dm

```diff
@@ -241,6 +241,9 @@
 	//Start now warning
 	var/start_now_confirmation = 0
 
+	// Allow observers before initialization
+	var/observer_initialize = 0
```

Comment:
I realise that this and line 735 should be `FALSE` and `TRUE` since it's a boolean, but all the other variables in this file are `1`s and `0`s so it'll look a bit out of place if I change it. And I don't particularly want to go through 917 lines to change everything manually.